LDAR_Sim/src/file_processing/output_processing/multi_simulation_outputs.py

```diff
@@ -0,0 +1,213 @@
+import os
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import re
+from file_processing.output_processing.output_utils import (
+    TIMESERIES_COL_ACCESSORS as tca,
+    EMIS_DATA_COL_ACCESSORS as eca,
+)
+
+
+class TS_SUMMARY_COLUMNS_ACCESSORS:
+    PROG_NAME = "Program Name"
+    SIM = "Simulation"
+    AVG_T_DAILY_EMIS = 'Average "True" Daily Emissions (Kg Methane)'
+    T_DAILY_EMIS_95 = '95th Percentile "True" Daily Emissions (Kg Methane)'
+    T_DAILY_EMIS_5 = '5th Percentile "True" Daily Emissions (Kg Methane)'
+    AVG_DAILY_COST = "Average Daily Cost ($)"
+    DAILY_COST_95 = "95th Percentile Daily Cost ($)"
+    DAILY_COST_5 = "5th Percentile Daily Cost ($)"
+
+
+class EMIS_SUMMARY_COLUMNS_ACCESSORS:
+    PROG_NAME = "Program Name"
+    SIM = "Simulation"
+    T_TOTAL_EMIS = 'Total "True" Emissions (Kg Methane)'
+    AVG_T_EMIS_RATE = 'Average "True" Emissions Rate (g/s)'
+    T_EMIS_RATE_95 = '95th Percentile "True" Emissions Rate (g/s)'
+    T_EMIS_RATE_5 = '5th Percentile "True" Emissions Rate (g/s)'
+    T_AVG_EMIS_AMOUNT = '"True" Average Emissions Amount (Kg Methane)'
+    T_EMIS_AMOUNT_95 = '95th Percentile "True" Emissions Amount (Kg Methane)'
+    T_EMIS_AMOUNT_5 = '5th Percentile "True" Emissions Amount (Kg Methane)'
+
+
+TS_SUMMARY_COLUMNS = [
+    TS_SUMMARY_COLUMNS_ACCESSORS.PROG_NAME,
+    TS_SUMMARY_COLUMNS_ACCESSORS.SIM,
+    TS_SUMMARY_COLUMNS_ACCESSORS.AVG_T_DAILY_EMIS,
+    TS_SUMMARY_COLUMNS_ACCESSORS.T_DAILY_EMIS_95,
+    TS_SUMMARY_COLUMNS_ACCESSORS.T_DAILY_EMIS_5,
+    TS_SUMMARY_COLUMNS_ACCESSORS.AVG_DAILY_COST,
+    TS_SUMMARY_COLUMNS_ACCESSORS.DAILY_COST_95,
+    TS_SUMMARY_COLUMNS_ACCESSORS.DAILY_COST_5,
+]
+
+EMIS_SUMMARY_COLUMNS = [
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.PROG_NAME,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.SIM,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.T_TOTAL_EMIS,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.AVG_T_EMIS_RATE,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.T_EMIS_RATE_95,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.T_EMIS_RATE_5,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.T_AVG_EMIS_AMOUNT,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.T_EMIS_AMOUNT_95,
+    EMIS_SUMMARY_COLUMNS_ACCESSORS.T_EMIS_AMOUNT_5,
+]
+
+
+def get_mean_val(df: pd.DataFrame, column: str) -> float:
+    return df[column].mean()
+
+
+def get_sum(df: pd.DataFrame, column: str) -> float:
+    return df[column].sum()
+
+
+# Currently defined to compute percentiles using a median-unbiased method as described in:
```

Comment:
Could a TODO be added here to add this comment also to the UserManual in the relevant section?

Reply:
Will do